File: sdkwork-app-sdk-go/types/plus_api_result_map_string_object.go
package types

// API调用结果
type PlusApiResultMapStringObject struct {
	Data map[string]interface{} `json:"data"`
	Code string `json:"code"`
	Msg string `json:"msg"`
	RequestId string `json:"requestId"`
	Ip string `json:"ip"`
	Hostname string `json:"hostname"`
	ErrorName string `json:"errorName"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/plus_api_result_map_string_object.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultMapStringObject:
    """API调用结果"""
    data: Dict[str, Any]
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str
